### PR TITLE
chore(docs): reformat tutorials part 1-8

### PR DESCRIPTION
**Pull request by muescha**
## Description

applied some formatting like in #21988 part zero to other parts

changes:
- note sections formatted the same
- hints sections formatted the same
- added some hints
- changed order of example files to be more logical (part 4)

## Related Issues

- #21988 `chore(docs): reformat tutorial - part zero`

### CODE REVIEW

File: docs/tutorial/part-one/index.md [hunk 1]
```diff
@@ -169,7 +171,7 @@ export default () => (
 
 ![New about page](05-about-page.png)
 
-Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `/about`.
+Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `http://localhost:8000/about`.
```
Comment (snitin315) [inline docs/tutorial/part-one/index.md]:
```suggestion
Just by putting a React component in the `src/pages/about.js` file, you now have a page accessible at `[http://localhost:8000/about](http://localhost:8000/about)`.
```

Comment (muescha) [reply docs/tutorial/part-one/index.md]: sorry - local links which results in 404 if not running should be only in code blocks and not be links

see: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#mark-localhost-urls-as-code-strings

Comment (snitin315) [reply docs/tutorial/part-one/index.md]: My bad :-)

**Comment by marcysutton** [comment]
@muescha thanks for the PR! To align this better with Gatsby PR requirements, can you please do two things:
1. open an issue to discuss the changes you want to make before opening a PR
2. open tightly-scoped PRs for each chunk of work, such as a section of the tutorial, to make it easier for us to review and test

Those steps will make this a lot easier on our team, and will follow our process for issues and PRs. I'm going to close this PR to reflect that, so let us know if you have any questions. And thanks for contributing to Gatsby!

**Pull request closed without merging**